src/lumigo_opentelemetry/instrumentations/fastapi/parsers/__init__.py

```diff
@@ -44,7 +44,7 @@ async def new_otel_receive():  # type: ignore
                 with instance.tracer.start_as_current_span("receive_body") as send_span:
                     send_span.set_attribute(
                         "http.request.body",
-                        dump_with_context("requestBody", return_value),
+                        dump_with_context("requestBody", return_value.get("body")),
```

Comment:
Are we sure that `return_value` can never be `None` and if so, will be a `dict`? I think it's worth being defensive there in case we can't be 100% sure